lib/content-guides-generator/index.js

```diff
@@ -23,24 +20,6 @@ const VersionsSerializer = new Serializer('version', {
 
 const versions = yaml.safeLoad(readFileSync('node_modules/@ember/guides-source/versions.yml', 'utf8'));
 
-let premberVersions = [...versions.allVersions, 'release'];
```

Comment:
why was this removed and what's the impact?

Reply:
I didn't want this running every time, so I removed it. We can add it back when we bring back prember

Reply:
@sivakumar-kailasam I'm a bit puzzled about the removal of prember here as well. Could you discuss what prompted its removal here? Saw something briefly in the PR description, but didn't understand the reasoning behind it ...